apis/placement/v1beta1/clusterresourceplacement_types.go

```diff
@@ -450,6 +452,72 @@ type ApplyStrategy struct {
 	ServerSideApplyConfig *ServerSideApplyConfig `json:"serverSideApplyConfig,omitempty"`
 }
 
+// ConflictResolutionConfig describes the action used to resolve conflicts if the resource
+// to be placed already exists in the target cluster.
+type ConflictResolutionConfig struct {
```

Comment:
ConflictResolutionConfig describes "when to apply" while ApplyStrategyType describes "how to apply".

"Recreate" should be treated as new "ApplyStrategyType"? it always delete the existing one before creating.

Reply:
good point